generate.sh

```diff
@@ -27,10 +27,9 @@ usage() {
     echo "  Either --pkg or --pkgCsv must be specified"
     echo ""
     echo "options:"
-    echo "  --pkg <packageName,version[,tfm]>  A package and version, no spaces, separated by comma.  If the optional"
-    echo "                                       tfm is specified, it will be used in the project that restores the"
-    echo "                                       package."
-    echo "                                       Example: System.Text.Json,4.7.0"
+    echo "  --pkg <packageName,version[,tfm]>  A package and version, no spaces, separated by comma.  If optional semicolon-separated"
```

Comment:
Consider renaming `tfm` to `tfms` as it betters calls out the support for multiples.

Reply:
Good point, thanks. Renamed